packages/cashier/src/stores/useCashierStores.tsx

```diff
@@ -6,7 +6,6 @@ const CashierStoreContext = createContext<CashierStore | null>(null);
 
 export const CashierStoreProvider: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
     const { modules } = useStore();
-    // const memoizedValue = useMemo(() => new CashierStore(), []);
```

Comment:
@amir-deriv Either keep it or remove the one at line `12` too 🙏🏻

This comment is a reference for the next person to know that we need to instantiate `CashierStore` here once the dependencies to `RootStore` are removed. If you decide to keep it maybe we can add some comments to describe this 🙇🏻

Reply:
Sonarcloud is complaining about this, since the scope of this card is to fix those issues, that's why I did this. since you think that this comment is important, I will revert this

Reply:
@amir-deriv Again, I don't think just getting a list of errors and then fixing them without knowing why we have those errors in the first place is a good approach to having better code quality.

Reply:
Yeah, I agree with you @farzin-deriv 

Also, I think these kind of errors must not be entered in the codebase from start
![image](https://github.com/binary-com/deriv-app/assets/129206554/17a4b27f-e330-43e6-a31a-72756329207b)


see the screenshot for reference, Sonarcloud rules are strict, (some rules are useless too and we should disable is like using the index as key), we can instead disable useless rules and check these on pre push hook with `sonarlint` 

cc: @ali-hosseini-deriv